contracts/AelinUpFrontDeal.sol

```diff
@@ -6,8 +6,6 @@ import "./MinimalProxyFactory.sol";
 import {SafeERC20} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
 import {IERC20} from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
 import "@openzeppelin/contracts/utils/cryptography/MerkleProof.sol";
-import {AelinDeal} from "./AelinDeal.sol";
```

Comment:
@0xlinus not sure why these files were here

Reply:
hmmm that was me probably